packages/x-components/src/x-modules/queries-preview/events.types.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Dictionary of the events of QueriesPreview XModule, where each key is the event name, and the
+ * value is the event payload type or `void` if it has no payload.
+ *
+ * @public
+ */
+export interface QueriesPreviewXEvents {}
```

Comment:
Import this in `src/wiring/events.types.ts`.

Reply:
Done!